packages/apps/auth0-database-scripts/src/get_user.ts

```diff
@@ -24,6 +25,11 @@ async function getUser(email: string): Promise<Auth0User | undefined> {
     return patronRecordToUser(patronRecord);
   } else if (patronRecordResponse.status === ResponseStatus.NotFound) {
     return undefined;
+  } else if (patronRecordResponse.status === ResponseStatus.DuplicateUsers) {
+    throw new ValidationError(
```

Comment:
If I change this to an `Error` I get the generic "Unknown error occurred" red box from Auth0. 🤷‍♀️